Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -155,6 +167,37 @@ click: function () { closeClicked.raiseEvent(this); }');
         },
 
         /**
+         * Gets or sets the security settings for iFrames generated by the Infobox.
+         * This will disable sandboxing for the iFrame, and allow HTML to be in the title.
+         * Only do this if you are in complete control of the data that will be loaded into the InfoBox,
+         * otherwise it is a security issue.
+         * @memberof InfoBox.prototype
+         *
+         * @type {Boolean}
+         */
+        disableSecurity : {
+            get : function() {
+                return this._disableSecurity;
+            },
+            set : function(value) {
+                this._disableSecurity = value;
+                var e = this._element.getElementsByClassName('cesium-infoBox-title')[0];
+                if (value) {
+                    e.removeAttribute('data-bind');
+                    e.setAttribute('data-bind', 'html: titleText');
+                    this._frame.removeAttribute('sandbox');
+                }
+                else {
+                    e.removeAttribute('data-bind');
+                    e.setAttribute('data-bind', 'text: titleText');
+                    this._frame.setAttribute('sandbox', 'allow-same-origin allow-popups allow-forms');
+                }
+                knockout.cleanNode(this._element);
```

Comment:
@ggetz I'm not sure that is possible, because I'm changing between the html and text name bindings. I'm not changing the actual value that they hold, which I believe is what `with` is used for